src/components/audit/filters/action.vue

```diff
@@ -34,6 +34,7 @@ export default {
       required: true
     }
   },
+  emits: ['input'],
```

Comment:
Could you change all instances of `'input'` in the `emits` option to the (somewhat verbose&hellip;) `'update:modelValue'` (which `v-model` uses in Vue 3)?

Reply:
done!